src/services/beerService.tsx

```diff
@@ -1,52 +1,54 @@
 import firebase from "../utils/firebaseConfig";
 
+export type Beer = {
+  id: string,
+  name: string, 
+  style: string,
+  brewery: string
+};
+
 const beerService = {
-  findAll: () => {
+  findAll: (): Array<Beer> => {
     const beersDB = firebase.database().ref("beersDB");
-    let list = [];
+    let list: (object)[] = [];
     beersDB.on("value", (snapshot) => {
       let previousList = snapshot.val();
       for (let id in previousList) {
         list.push({ id, ...previousList[id] });
       }
     });
-    return list;
+    return list as Array<Beer>;
```

Comment:
we should not force types on objects we control, we should better make objects with controlled object with the right types as fast as possible : 

``` javascript
const makeBeer: Beer = (id: string, remoteBeer: any /*Here we can use any because we actually use an object from a remote service */) => {
  /*
    here we can simply assume that the object is good and return {id, name: remoteBeer.name}... or make error handling if we want. For now I don't think it's necessary
  */
};

//..

let list: Beer[] = [];
beersDB.on(...
  list.push(makeBeeri(id, previousList[id]));
);
return list;
```